src/Features/CSharp/Portable/Debugging/DataTipInfoGetter.cs

```diff
@@ -39,11 +39,11 @@ internal static async Task<DebugDataTipInfo> GetInfoAsync(Document document, int
 
             if (expression is LiteralExpressionSyntax)
             {
-                // If the user hovers over a literal, give them a DataTip for the type of the
-                // literal they're hovering over.
-                // Partial semantics should always be sufficient because the (unconverted) type
-                // of a literal can always easily be determined.
-                var (_, semanticModel) = await document.GetFullOrPartialSemanticModelAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
removed this extension as the main WithFrozenPartialSemantics extension has these semantics now.